CMakeLists.txt

```diff
@@ -327,10 +331,11 @@ endif()
 ###################################################################################################
 
 if(CUTLASS_ENABLE_SYCL)
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -O3 -fsycl -ffast-math -mllvm -enable-global-offset=false -fsycl-targets=nvptx64-nvidia-cuda -Xsycl-target-backend")
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} --cuda-gpu-arch=sm_${CUTLASS_NVCC_ARCHS}")
-  set(CMAKE_CUDA_ARCHITECTURES "${CUTLASS_NVCC_ARCHS}")
-  add_definitions( -DCUTLASS_ENABLE_SYCL=${CUTLASS_ENABLE_SYCL} )
+  set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
+
+  add_compile_definitions(CUTLASS_ENABLE_SYCL)
+  set(DPCPP_SYCL_TARGET "nvptx64-nvidia-cuda" CACHE STRING "Default target for Nvidia GPU")
+  find_package(DPCPP)
```

Comment:
Why use this approach?
Isn't the BKM to use find_package(IntelSYCL REQUIRED) (https://www.intel.com/content/www/us/en/docs/dpcpp-cpp-compiler/developer-guide-reference/2024-1/use-cmake-with-the-compiler.html)